package.json

```diff
@@ -251,10 +250,13 @@
     "velocity-react": "^1.3.3",
     "webpack": "^4.46.0",
     "webpack-bundle-analyzer": "^4.5.0",
-    "webpack-cli": "^4.9.1",
-    "webpack-dev-server": "3.11.2",
+    "webpack-cli": "^4.9.2",
+    "webpack-dev-server": "3.11.3",
     "webpack-manifest-plugin": "^3.0.0",
     "whatwg-fetch": "2.0.3",
     "xhr": "^2.5.0"
+  },
+  "devDependencies": {
+    "standard": "12.0.1"
```

Comment:
Note: `standard` needs to be in `devDependencies` due to some current `eslint` config expectations. Still nothing else should go here.